index.html

```diff
@@ -1,120 +1,138 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-  </head>
-
-  <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-      <header id="header">
-        <img  src='https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png' id="header-img">
-        <nav id="nav-link">
-          <ul>
-            <li><a>Features</a> </li>
-            <li><a>How It Works</a> </li>
-            <li><a>Pricing</a> </li>
-      
-          </ul>
-        </nav>
-      </header>
-      <h2>Handcrafted, home-made masterpieces</h2>
-      <form action="" id='form'>
-        <input type="search" name="search" id="search" placeholder="Enter your email address">
-        <button class='btn'> <span class="bolder"> GET STARTED</span> </button>
-      </form>
- 
-      
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
In my opinion, the title should be more descriptive.

Reply:
- [ ] Please provide an explanation to your request so that the student can understand why is it important to change it 😄 🙏

Reply:
you have done very well so far, I would like you to add a more descriptive title as it will help users to understand what this page is all about.